apps/web/lib/apps/routing-forms/[...pages]/getServerSideProps.ts

```diff
@@ -53,7 +45,7 @@ export async function getServerSideProps(
 
   return {
     props: {
-      appUrl: app.simplePath || `/apps/routing-forms`,
```

Comment:
app.simplePath is always `/routing`, and we don't need to call `await getAppWithMetadata({ slug: "routing-forms" })` for this info